src/lender/assessor/base.sol

```diff
@@ -201,6 +216,8 @@ contract BaseAssessor is Math, Auth {
             return true;
         }
 
+        _drip();
```

Comment:
same for redeem approve

Reply:
I think it would make sense to make this function view as well

Reply:
*note